### ISSUE TITLE
FR: Commandline examples 

### ISSUE THREAD

**Issue by irregularisnotrandom** [duplicati/duplicati#1547]
Hi

It would be nice if there are some commandline examples for testing, verifying etc.

For instance I am trying to test a backup but I am getting this bizarre message. I can't tell if I am making a mistake with the line or if there is a problem with the code.

The backup is on a samba drive on Linux.

"An attempt was made to load an assembly from a network location which would have caused the assembly to be sandboxed in previous versions of the .NET Framework. This release of the .NET Framework does not enable CAS policy by default, so this load may be dangerous. If this load is not intended to sandbox the assembly, please enable the loadFromRemoteSources switch. See http://go.microsoft.com/fwlink/?LinkId=155569 for more information."

**Comment by kenkendk** [comment]
The error mentioned here and in #1548 is most likely the windows file protection feature:
https://superuser.com/questions/38476/this-file-came-from-another-computer-how-can-i-unblock-all-the-files-in-a

tl;dr: Delete the extracted folder, right click the zip file, click properties, click "Unblock", click ok, extract zip archive again.

**Comment by kenkendk** [comment]
For the examples, we can add them here:
https://github.com/duplicati/duplicati/blob/master/Duplicati/CommandLine/help.txt#L43

If you post an example command, I can see if it looks right.

**Comment by irregularisnotrandom** [comment]
I do not have an unblock button or a way to unblock the *.zip.aes files that  are on the network.. I was just trying to verify the Duplicati backups from the command line. 

I will see what MS solution is later.

**Comment by kenkendk** [comment]
I do not think the solutions will work.

Perhaps you are trying to run the Duplicati code from a network drive, which may make Windows block the execution. Try storing the files on your local disk.

**Comment by kenkendk** [comment]
I assume this was resolved or no longer interesting, feel free to re-open.
